server/sonar-web/src/main/js/apps/settings/utils.js

```diff
@@ -84,7 +84,7 @@ export function getEmptyValue(definition) {
 
   if (definition.type === TYPE_PROPERTY_SET) {
     const value = {};
-    definition.fields.forEach(field => value[field.key] = getEmptyValue(field));
+    definition.fields.forEach(field => (value[field.key] = getEmptyValue(field)));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Extract the assignment of "value[field.key]" from this expression. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AAssignmentWithinCondition)